### DISCUSSION TITLE
interested in creating an AI voice cloning tool

### DISCUSSION THREAD

**Discussion by 100Khushi** [community/community#78014] [Programming Help]
Hello,
I'm interested in creating an AI voice cloning tool similar to Wavel's [voice cloning](https://wavel.ai/studio/ai-voice-cloner) tool. Can anyone provide guidance on the key steps, algorithms, or frameworks involved in developing such a tool? Any insights or resources to get started on this project would be highly appreciated.
Thanks in advance :)

**Comment by samus-aran** [comment]
Hi @100Khushi , Thanks for joining us for the GitHub Universe celebrations in the GitHub Community, we're glad you're here! 

We want to make sure you’re getting the best support you can now that GitHub Universe has ended so I've gone ahead and moved this to the most relevant category for you.

Thank you for helping us maintain a productive and tidy community for all our members.
